src/sidebar.cpp

```diff
@@ -305,28 +305,39 @@ void player::disp_status( const catacurses::window &w, const catacurses::window
 
     if( !style.empty() ) {
         const auto style_color = is_armed() ? c_red : c_blue;
-        const int x = sideStyle ? ( getmaxx( weapwin ) - 13 ) : 0;
-        mvwprintz( weapwin, 1, x, style_color, style );
+        const int x = sideStyle ? ( getmaxx( weapwin ) - 13 ) : ( getmaxx( weapwin ) - 12 );
+        mvwprintz( weapwin, 0, x, style_color, style );
     }
 
-    wmove( w, sideStyle ? 1 : 2, 0 );
+    std::string hunger_string = "";
+    nc_color hunger_color = c_yellow;
     if( get_hunger() >= 300 && get_starvation() > 2500 ) {
-        wprintz( w, c_red,    _( "Starving!" ) );
+        hunger_color = c_red;
+        hunger_string = "Starving!";
```

Comment:
Why remove _(translation) here and at the next lines?

Reply:
sorry aboutthat, i had no idea this was related to translation..